vscode-dotnet-runtime-library/src/Acquisition/DotnetConditionValidator.ts

```diff
@@ -247,6 +228,45 @@ Please set the PATH to a dotnet host that matches the architecture ${requirement
         }
     }
 
+    private getFeatureBand(availableVersion: string, requirement: IDotnetFindPathContext): number | null
+    {
+        const availableBandStr: string | null = requirement.acquireContext.mode === 'sdk' ?
+            (() =>
+            {
+                const featureBand = versionUtils.getFeatureBandFromVersion(availableVersion, this.workerContext.eventStream, this.workerContext, false);
+                if (featureBand)
+                {
+                    return featureBand;
+                }
+                return null;
+            })() : null;
+        return availableBandStr ? Number(availableBandStr) : null;
+    }
+
+    private getPatchOrFeatureBandWithPatch(availableVersion: string, requirement: IDotnetFindPathContext): number | null
+    {
+        const availablePatchStr: string | null = requirement.acquireContext.mode !== 'sdk' ?
+            versionUtils.getRuntimePatchVersionString(availableVersion, this.workerContext.eventStream, this.workerContext)
+            :
+            (() =>
+            {

```

Comment:
nit: Is this style intended?

Reply:
It is auto-formatted within the style guidelines set by the project, so in a way, yes..
even though I agree it looks ugly.